src/main/resources/db/migration/V1_51__add_target_pik_to_fund_transfer_exchange.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE fund_transfer_exchange
+  ADD COLUMN target_pik VARCHAR NULL;
```

Comment:
Would be great to have some kind of a limit on this column length. Is PIK an IBAN code? In that case it's up to 34 characters: https://en.wikipedia.org/wiki/International_Bank_Account_Number
Ie. `VARCHAR(34)`